src/components/sq-modal-profile-cache/sq-modal-profile-cache.component.tsx

```
@@ -0,0 +1,236 @@
+        ),
+      }),
+      {},
+    ) as Record<CacheField, string>,
+  )
+
+  const resetState = useCallback(
+    () =>
+      setState(
+        fieldsCache?.reduce(
+          (acc, field) => ({
+            ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)